src/tools/tidy/src/unit_tests.rs

```diff
@@ -56,7 +56,8 @@ pub fn check(root_path: &Path, bad: &mut bool) {
             let line = line.trim();
             let is_test = || line.contains("#[test]") && !line.contains("`#[test]");
             let is_bench = || line.contains("#[bench]") && !line.contains("`#[bench]");
-            if !line.starts_with("//") && (is_test() || is_bench()) {
+            let manual_skip = line.contains("//tidy:skip");
+            if !line.starts_with("//") && (is_test() || is_bench()) && !manual_skip {
```

Comment:
Would it be possible to deny or ignore `//tidy:skip` inside libcore and liballoc? Even if you were to use `//tidy:skip` inside either of those the test simply wouldn't get run ever.